docs/java-api/docs/update-by-query.asciidoc

```diff
@@ -41,78 +36,50 @@ The `UpdateByQueryRequestBuilder` API supports filtering the updated documents,
 limiting the total number of documents to update, and updating documents
 with a script:
 
-[source,java]
---------------------------------------------------
-UpdateByQueryRequestBuilder updateByQuery = UpdateByQueryAction.INSTANCE.newRequestBuilder(client);
 
-updateByQuery.source("source_index")
-        .filter(termQuery("level", "awesome"))
-        .size(1000)
-        .script(new Script("ctx._source.awesome = 'absolutely'", ScriptType.INLINE, "painless", emptyMap()));
-
-BulkByScrollResponse response = updateByQuery.get();
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{client-reindex-tests}/ReindexDocumentationIT.java[update-by-query2]
 --------------------------------------------------
 
 `UpdateByQueryRequestBuilder` also enables direct access to the query used
 to select the documents. You can use this access to change the default scroll size or
 otherwise modify the request for matching documents.
 
-[source,java]
+["source","java",subs="attributes,callouts,macros"]
 --------------------------------------------------
-UpdateByQueryRequestBuilder updateByQuery = UpdateByQueryAction.INSTANCE.newRequestBuilder(client);
-
-updateByQuery.source("source_index")
-        .source().setSize(500);
-
-BulkByScrollResponse response = updateByQuery.get();
+include-tagged::{client-reindex-tests}/ReindexDocumentationIT.java[update-by-query3]
 --------------------------------------------------
 
 You can also combine `size` with sorting to limit the documents updated:
 
-[source,java]
+["source","java",subs="attributes,callouts,macros"]
 --------------------------------------------------
-UpdateByQueryRequestBuilder updateByQuery = UpdateByQueryAction.INSTANCE.newRequestBuilder(client);
-
-updateByQuery.source("source_index").size(100)
-        .source().addSort("cat", SortOrder.DESC);
-
-BulkByScrollResponse response = updateByQuery.get();
+include-tagged::{client-reindex-tests}/ReindexDocumentationIT.java[update-by-query4]
```

Comment:
I'd probably name these `update-by-query-size` and `update-by-query-scroll-size` and the like.